src/posts/official-ppa-20230206.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: "/news/official-ppa-20230206"
+date: "2023-02-06"
+title: "Announcing Apptainer Official PPA for Ubuntu"
+---
+
+We are pleased and excited to announce that Apptainer official PPA for Ubuntu 
```

Comment:
I think it's going a bit far to say excited for this announcement. I'd like you to remove "and excited" and change the exclamation point to a period.

Reply:
Thank you, I'm preparing to update PR right now.